charon/tests/ui.rs

```diff
@@ -0,0 +1,177 @@
+//! Ui tests for the charon compiler. Each `<file>.rs` file in `ui/` will be passed to the charon
+//! driver. The corresponding pretty-printed llbc output will be stored in `<file>.llbc`, and CI
+//! will ensure these stay up-to-date.
+//!
+//! Files can start with special comments that affect the test behavior. Supported magic comments:
+//! see `HELP_STRING`.
```

Comment:
I think there is a different syntax to refer to a definition in a comment? Something like [HELP_STRING]?...